### PR TITLE
Add migration script for product export job parameters

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: UPGRADE-1.6.md [hunk 1]
```diff
@@ -275,6 +275,58 @@ my_bundle.mass_edit_action.my_action:
...
+            data: []
+            structure:
+                scope: mobile
+                locales:
+                    - fr_FR
+                    - en_US
+                    - de_DE
+```
+
+You can now notice a `filter` key containing `data` and `structure`.
```
Comment (jjanvier) [inline UPGRADE-1.6.md]: filters

File: upgrades/schema/orm/JobInstanceParametersMigration.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+use Doctrine\ORM\EntityNotFoundException;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+
+/**
+ * @author    Adrien Pétremann <adrien.petremann@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class JobInstanceParametersMigration
```
Comment (jjanvier) [inline upgrades/schema/orm/JobInstanceParametersMigration.php]: no possibility to use a standard doctrine migration?

**Comment by jjanvier** [comment]
gtm once fixed

File: upgrades/schema/orm/JobInstanceParametersMigration.php (hunk 2 above)
Comment (grena) [inline upgrades/schema/orm/JobInstanceParametersMigration.php]: Unfortunately no, this migration needs a bit of logic impossible to achieve with a simple Doctrine migration

File: UPGRADE-1.6.md [hunk 3]
```diff
@@ -275,6 +275,58 @@ my_bundle.mass_edit_action.my_action:
...
+            structure:
+                scope: mobile
+                locales:
+                    - fr_FR
+                    - en_US
+                    - de_DE
+```
+
+You can now notice a `filters` key containing `data` and `structure`.
+- `data`: Restrict rows to export
```
Comment (willy-ahva) [inline UPGRADE-1.6.md]: Restricts

File: upgrades/schema/orm/JobInstanceParametersMigration.php (hunk 2 above)
Comment (willy-ahva) [inline upgrades/schema/orm/JobInstanceParametersMigration.php]: Use the container aware

**Comment by momoss** [comment]
GTM

**Comment by willy-ahva** [comment]
GTM

**Pull request merged**